### DISCUSSION TITLE
Bitbucket server to github enterprise migration

### DISCUSSION THREAD

**Discussion by Bhaagi541** [community/community#129192] [Enterprise]
### Body

Hi I am trying to migrate repo from bitbucket server which is on Linux to github enterprise account in cloud.the repo in bitbucket having 5gb size and pull request .need to migrate everything .could you please suggest the process

**Comment by queenofcorgis** [comment]
Hi there,

We have documentation on that, please check out [this doc](https://docs.github.com/en/migrations/using-github-enterprise-importer/migrating-from-bitbucket-server-to-github-enterprise-cloud/overview-of-a-migration-from-bitbucket-server-to-github-enterprise-cloud) for an overview of a migration from Bitbucket Server

**Comment by Bhaagi541** [comment]
Hi I need the direct and complete process if any to migrate repositories,and associated meta data(pull request, history,issues and wikis ) to github from bitbucket.could you please help me on this Thanks in advance!Thanks,BhaagyaOn Jun 21, 2024 6:54 PM, Lili ***@***.***> wrote:
Hi there,
We have documentation on that, please check out this doc for an overview of a migration from Bitbucket Server

**Reply by queenofcorgis** [reply]
@Bhaagi541 our documentation will have all those details!

**Comment by Bhaagi541** [comment]
Could you please share the process document like what are the steps need to follow, pre-requisites..I don't wanna use S3 bucket or azure storage account as intermediate storage ..I wanna migrate directly from bitbucket to github..and the bitbucket repo contains 7 gb size.and also wanna migrate pull request ,wikis etc..how can I migrate all this meta data like pull request,issues etc.., could please share the straight away url.And if any scripts available in powershell or bashThanks in advance!Thanks,Bhagya laxmi On Jun 24, 2024 6:59 PM, Lili ***@***.***> wrote:
@Bhaagi541 our documentation will have all those details!
